src/ui/components/login/loginComponent.ts

```diff
@@ -6,32 +6,84 @@ const Vue = <vuejs.VueStatic>require("vue");
 
 import "../../css/login.less";
 
+import CDNService from "../../services/cdnService.ts";
+import StaticService from "../../services/staticService.ts";
+import NetworkService from "../../services/networkService.ts";
+import SettingService from "../../services/settingService.ts";
+import ModalComponent from "../modal/modalComponent.ts";
+
 @Component({
     template: require("./loginView.html")
 })
 export default class LoginComponent extends Vue {
     private isLoading: boolean;
     private previousInvalid: boolean;
-    
+
     private username: string;
-    private password: string;
-    
+    private host: string;
+    private path: string;
+    private port: string;
+    private state: string;
+    private icon: any;
+
     data() {
+        if (localStorage.getItem("path") != undefined && localStorage.getItem("path") != "") {
```

Comment:
`this.path = localStorage.getItem("path") || "Path to LoL 4.20";`

Reply:
d